lib/snapshot.go

```diff
@@ -519,6 +519,7 @@ func (snap *Snapshot) Stop() {
 	// It's important!!!
 }
 
+// DEPRECATED: This function is no longer used in PoS.
```

Comment:
Looks like this is no longer used anywhere, can we just delete this whole function?